scripts/generate_snos.sh

```diff
@@ -11,7 +11,7 @@ set -e
 SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 REPO_ROOT=$( dirname -- $SCRIPT_DIR )
 
-CAIRO_VERSION="0.13.2.1"
+CAIRO_VERSION="0.13.3"
 COMPILER_VERSION="0.13.2"
```

Comment:
We might as well update `COMPILER_VERSION` here too?

Honestly I originally set them to these values in #7 only to ensure the generated files match exactly with the legacy LFS files - as proof that the scripts work correctly. Now there's no reason to keep using older versions.

Reply:
Yep, I didn't bump also before, but worked fine, tested on SNOS and layout bridge proof, both have been verified correctly.